src/adapters/scene.js

```diff
@@ -145,10 +147,29 @@ Scene.prototype.initMapBox = function() {
       }, this.DOUBLE_TAP_DELAY_MS);
     });
 
+    // Long touch on mobile (500ms)
+    // Custom implementation because the contextmenu event isn't supported by MapBox.
+    // Long touch is initiated with a touchstart event
+    // It is canceled if many fingers touch the map, or if one of these events is initiated:
+    // touchmove, touchcancel, touchend, pointerdrag, pointermove, moveend, gesturestart, gesturechange, gestureend
     if (isMobileDevice()) {
```

Comment:
I think we should get rid of this condition. Listening to touch-specific events should be enough. And that way we'll support big screens (not considered as mobile) with touch support.
More generally, we should use `isMobileDevice` only when there aren't more reliable alternatives.